### PR TITLE
PXB-2490 xtrabackup privileges update (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   source/using_xtrabackup/privileges.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst [hunk 1]
```diff
@@ -119,6 +119,12 @@ backed up:
   to look up the ``innodb_to_lsn`` values in the
   :ref:`PERCONA_SCHEMA.xtrabackup_history <xtrabackup_history>` table.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst:120]:
```suggestion
``SELECT`` privilege in order to use keyring_file component in order for the feature to loop up installed keyring components in the :ref:`performance_schema.keyring_component_status <DONT KNOW WHAT TO ADD HERE :D >` table.
```

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst [hunk 2]
```diff
@@ -119,6 +119,12 @@ backed up:
...
 
+Using the keyring_file component requires the following privilege:
+
+.. sourcecode::
+
+   GRANT SELECT ON performance_schema.keyring_component_status TO bkpuser@'localhost' 
+
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst]: move to `An SQL example of creating a database user with the minimum privileges required
to full backups would be:`

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst:120]: Fixed.

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst]: OK.

File: storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst [hunk 3]
```diff
@@ -131,3 +130,10 @@ to full backups would be:
    mysql> GRANT BACKUP_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'bkpuser'@'localhost';
    mysql> GRANT SELECT ON performance_schema.log_status TO 'bkpuser'@'localhost';
    mysql> FLUSH PRIVILEGES;
+
+``SELECT`` privilege on the `keyring_component_status table <https://dev.mysql.com/doc/refman/8.0/en/performance-schema-keyring-component-status-table.html>`__  to view the attributes and status of the installed keyring component when in use.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst]: seems like we are missing a `*` in the begining

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst]: fixed

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**